packages/invoices/src/InboundShipment/DetailView/SidePanel/RelatedDocumentsSection.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { memo } from 'react';
+import {
+  Grid,
+  DetailPanelSection,
+  PanelField,
+  PanelLabel,
+  PanelRow,
+  useTranslation,
+  Tooltip,
+  Link,
+  useFormatDate,
+  RouteBuilder,
+} from '@openmsupply-client/common';
+import { AppRoute } from '@openmsupply-client/config';
+import { useInboundFields } from '../../api';
+
+export const RelatedDocumentsSectionComponent = () => {
+  const t = useTranslation('replenishment');
+  const d = useFormatDate();
+  const { requisition } = useInboundFields('requisition');
+
+  let tooltip = '';
+  if (requisition) {
+    const { user, createdDatetime } = requisition;
+    tooltip = t('messages.internal-order-created-on', {
+      date: d(new Date(createdDatetime)),
+    });
+    if (user?.username && user.username !== 'unknown') {
```

Comment:
that would be useful as a utility function - though the full thing would require the translation hook too, which is a pain.

Reply:
Hmm you're right it's a lot of duplication. What about a `UserTooltip`? Should that use the info icon or another? I'l make an issue